clang/include/clang/Sema/Sema.h

```diff
@@ -5187,7 +5187,9 @@ class Sema {
   enum BoundsDeclarationCheck {
       BDC_Assignment,
       BDC_Initialization,
-      BDC_Statement
+      BDC_Statement,
```

Comment:
Should the enums be listed alphabetically?

Reply:
I can make this adjustment. Just curious, is it a C++ coding convention to list items alphabetically in an enum?

Reply:
No, this is not a C++ coding convention. Listing enums/includes, etc alphabetically can help with code readability.